imm.yaml

```diff
@@ -3,29 +3,32 @@
     Id: 74e29f9b-6dca-4161-8fdd-b468a1eb185d
     Measures:
         -
-            Score: 0
-            Description: None
+            Score: 1
+            Description: Configured
 -
     Name: 'Coding Standards'
     Id: f6f6490f-9493-4dc3-a674-15584fa951d8
     Measures:
         -
-            Score: 0
-            Description: None
+            Score: 1
+            Description: 'Enforced via tooling'
```

Comment:
Not in the specs project though. Also things that we normally have configured to raise issues - e.g. missing `ConfigureAwait(false)` - are not present.

Reply:
Same issue as the previous point: we have enforced through tooling everything that it would be reasonable for us to enforce through tooling. Since there's nothing more to be done here, we're done, so we score the point.

Reply:
Having said that, set back to 0 as per previous point.